rules/DB_processing/format_n_train_classifiers.rules

```diff
@@ -1,35 +1,35 @@
 ## Rules to format the taxonomy into a format for DADA2 implementation of RDP.
-rule DADA2_prep_tax_db:
-    conda:
-        "../../envs/amplicons_r_utils.yml"
-    container:
-        singularity_envs["r_utils"]
-    input:
-        ref_seqs="{prefix}/QIIME/DB_amp.fasta",
-        ref_tax="{prefix}/QIIME/DB_amp_taxonomy.txt",
-    output:
-        King_to_Species="{prefix}/dada2rdp/DADA2_DB_amp_taxonomy_King_to_Species.txt",
-        King_to_Genus="{prefix}/dada2rdp/DADA2_DB_amp_taxonomy_King_to_Genus.txt",
-        Genus_species="{prefix}/dada2rdp/DADA2_DB_amp_taxonomy_Genus_species.txt",
-    log:
-        "{prefix}/logs/dada2rdp/DB_amp_taxonomy_dada2_prep.log",
-    threads: 1
-    script:
-        "scripts/dada2_prep_tax.R"
-
-
-rule Hash_DADA2:
-    input:
-        King_to_Species="{prefix}/dada2rdp/DADA2_DB_amp_taxonomy_King_to_Species.txt",
-        King_to_Genus="{prefix}/dada2rdp/DADA2_DB_amp_taxonomy_King_to_Genus.txt",
-        Genus_species="{prefix}/dada2rdp/DADA2_DB_amp_taxonomy_Genus_species.txt",
-    output:
-        hash="{prefix}/dada2rdp/DADA2_DB.hash",
-    threads: 1
-    shell:
-        """
-        md5sum {input.King_to_Species} {input.King_to_Genus} {input.Genus_species} | sort -k 2 | md5sum | cut -f 1 -d " " > {output}
-        """
+#rule DADA2_prep_tax_db:
```

Comment:
I'm against commenting stuff out. If it's not necessary anymore, delete it. If we need to find that code again later, git has all the history...